File: backend/src/main/java/com/example/demo/entities/Host.java
package com.example.demo.entities;

import java.sql.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@ToString
@Table(name = "hosts")
public class Host {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="hostId")
    private int host_id;
    
    @Column(name="fname")
    private String fname;
    
    @Column(name="lname")
    private String lname;
    
    @Column(name="emailId")
    private String email_id;
    
    @Column(name="contact")
    private String contact;
    
    @Column(name="dob")
    private Date dob;
    
    @Column(name="pancard_number")
    private String pancard_number;
    
    @Column(name="adharcard_number")
    private String adharcard_number;
    
    @Column(name="upi_id")
    private String upi_id;
    
    @Column(name="reg_date")
    private Date reg_date;

    @ManyToOne
    @JoinColumn(name = "area_id")
    private Area area;

    @ManyToOne
    @JoinColumn(name = "uid")
    private User user;
    
    @Column(name="address")
    private String address;

}

File: dot.net/BookMyCar/Models/Host.cs
using System;
using System.Collections.Generic;

namespace BookMyCar.Models;

public partial class Host
{
    public int HostId { get; set; }

    public int? Uid { get; set; }

    public string Fname { get; set; } = null!;

    public string Lname { get; set; } = null!;

    public int? AreaId { get; set; }

    public string EmailId { get; set; } = null!;

    public string Contact { get; set; } = null!;

    public DateOnly? Dob { get; set; }

    public string PancardNumber { get; set; } = null!;

    public string AdharcardNumber { get; set; } = null!;

    public string UpiId { get; set; } = null!;

    public DateOnly? RegDate { get; set; }

    public string? Address { get; set; }

    public virtual Area? Area { get; set; }

    public virtual ICollection<Car> Cars { get; set; } = new List<Car>();

    public virtual User? UidNavigation { get; set; }
}
